remove-webhook/adjust-downstream-webhook.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+usage() {
+  cat << EOF
+usage: bash adjust-downstream-webhook.sh [--insecure-skip-tls-verify]
+
+This script adjusts the rancher-webhook chart release in all clusters managed by Rancher (excluding the local cluster).
+Depending on the version of Rancher, it either deletes the downstream webhook release, adjusts its version and restarts, or does nothing.
+Requires kubectl and helm to be installed and available on \$PATH.
+
+RANCHER_URL without a trailing slash must be set with the server URL of Rancher.
+RANCHER_TOKEN must be set with an admin token generated with no scope.
+To ignore TLS verification, set --insecure-skip-tls-verify.
+
+Users also need to ensure they have the rancher-charts repo in the local Helm index.
+EOF
+}
+
+if [ "$1" == "-h" ]; then
+  usage
+  exit 0
+fi
+
+delete_webhook() {
+  cluster="$1"
+  current_chart=$(helm list -n cattle-system -l name=rancher-webhook | tail -1 | cut -f 6)
+  echo "Deleting $current_chart from cluster $cluster."
+  helm uninstall rancher-webhook -n cattle-system
```

Comment:
This will make the script exit non-zero if rancher-webhook isn't installed, which will be the case most of the time for <=2.6.13 and <=2.7.0. Consider appending `|| true` so that the script run can be considered a success even if there was nothing to uninstall.

Reply:
The script will exit with a non-zero still because it will reach the last line, which removes the temporary Kubeconfig, I think.
Also, users aren't supposed to run this script if they don't have an unnecessary webhook downstream. And the code won't attempt to delete the webhook given the wrong Rancher version.

Reply:
You're right, read it too fast, `-e` isn't set